database/schema.sql

```diff
@@ -0,0 +1,46 @@
+-- create database for catalog
+CREATE DATABASE IF NOT EXISTS catalog_of_things;
+
+-- Create authors table
+CREATE TABLE IF NOT EXISTS authors(
+  id serial not null primary key,
+  first_name varchar(50),
+  last_name varchar(50)
+);
+
+-- create games table
+CREATE TABLE IF NOT EXISTS games(
+  id serial not null primary key,
+  multiplayer boolean,
+  last_played date,
+  publish_date date
+
```

Comment:
- [ ] Here you forgot to close the `(` at the end of the query, kindly make sure to fix this so the code doesn't produce an error

```suggestion
CREATE TABLE IF NOT EXISTS games(
  id serial not null primary key,
  multiplayer boolean,
  last_played date,
  publish_date date
)

```